libs/design/switch/src/switch/switch.component.ts

```diff
@@ -7,14 +9,80 @@ import {
   Output,
 } from '@angular/core';
 
+import { DAFF_LOADING_ICON_COMPONENTS } from '@daffodil/design/loading-icon';
+import { DaffToastService } from '@daffodil/design/toast';
+
 @Component({
   selector: 'daff-switch',
   changeDetection: ChangeDetectionStrategy.OnPush,
   templateUrl: './switch.component.html',
   styleUrls: ['./switch.component.scss'],
   standalone: true,
+  imports: [
+    CommonModule,
+    DAFF_LOADING_ICON_COMPONENTS,
+  ],
 })
 export class DaffSwitchComponent {
   @Input() checked = false;
+  @Input() loading = false;
+  @Input() position: 'left' | 'right' | 'top' | 'bottom' = 'left';
```

Comment:
Refer to `DaffNavPlacement` on how we implement input properties like this. Also, update name to `labelPosition` for better clarity.